lib/update.js

```diff
@@ -0,0 +1,111 @@
+'use babel';
+
+import { updates } from 'titanium-editor-commons';
+import { log } from 'util';
```

Comment:
<!--
  0 failure: 
  1 warning:  lib/update.js lin...
  
  
  DangerID: danger-id-default;
  File: lib/update.js;
  Line: 4;
-->

- :warning: lib/update.js line 4 – 'log' is defined but never used. (no-unused-vars)